agent/agent.go

```diff
@@ -171,7 +171,7 @@ func (r *Agent) Run(ctx context.Context, recipe recipe.Recipe) (run Run) {
 	go func() {
 		defer func() {
 			if r := recover(); r != nil {
-				run.Error = fmt.Errorf("%s", r)
+				run.Error = fmt.Errorf("recover run panic: %s", r)
```

Comment:
could we change the error message to something more meaningful? something like `agent run: close stream: panic: %s`